### PR TITLE
Create Tab Component

### PR DESCRIPTION
**Pull request by viktor-ku**
Hi.

Welcome Tab Component :tada: 

Closes issue #21

### CODE REVIEW

**Comment by mariolamacchia** [comment]
👋 is this still WIP?

**Comment by kuroljov** [comment]
@mariolamacchia Yeah. Need tests and examples

**Comment by viktor-ku** [comment]
@mariolamacchia sorry that this PR takes that long. I will have more time dedicated to it.

File: src/modules/Tab/TabPane.jsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+  methods: {
+    findParent() {
+      let parent = this.$parent;
+      const name = Tab.name;
+
+      while (parent.$options.name !== name) {
+        parent = parent.$parent;
+      }
+
+      return parent;
```
Comment (mariolamacchia) [inline src/modules/Tab/TabPane.jsx]: I'd throw an exception if `TabPane` isn't inside a `Tab`

**Comment by viktor-ku** [comment]
@mariolamacchia there are tow things left to implement and I need to write some tests too. Even though you approved it I think I will finish this PR in this week and then we can merge it.

File: src/modules/Tab/TabPane.jsx (hunk 1 above)
Comment (viktor-ku) [reply src/modules/Tab/TabPane.jsx]: `findParent` shouldn't throw anything because it's just a helper function to find a `Tab` component

Comment (mariolamacchia) [reply src/modules/Tab/TabPane.jsx]: What I'm saying is that if someone doesn't put TabPane under Tab, the error will probably be something like `Cannot find addTab of undefined`. I'd prefer a more descriptive message

Comment (viktor-ku) [reply src/modules/Tab/TabPane.jsx]: Yeah, right. Will do

**Comment by viktor-ku** [comment]
@mariolamacchia all features that `react-semantic-ui` tabs has are implemented now. Plus `disabled` state for the `sui-tab-pane` as a bonus. All I need to do now is to write some tests. Feels like it's close to the finish :cake:

**Comment by viktor-ku** [comment]
@mariolamacchia You can go ahead and merge this PR because I don't know how much time I will spend on these tests. However if you want tests then let me know.

**Comment by viktor-ku** [comment]
@ArStah merge maybe?

**Comment by mariolamacchia** [comment]
Sorry I missed your message. Looks great, thanks a lot for your help! ❤️

File: src/modules/Tab/TabPane.jsx (hunk 1 above)
Comment (alpeshgit) [reply src/modules/Tab/TabPane.jsx]: > I'd throw an exception if `TabPane` isn't inside a `Tab`

I get this error while creating dynamic tabs using v-for. Any work around for this?

**Pull request merged**